server/app.js

```diff
@@ -4,7 +4,7 @@ const path = require("path");
 var cookieParser = require("cookie-parser");
 var logger = require("morgan");
 
-const passport = require("passport");
+// const passport = require("passport");
```

Comment:
Since this isn't used here, but in the login strategies where it is imported anyways, this can probably be removed.